### DISCUSSION TITLE
Super noob here - not seeing where to convert PDF to markdown in the documentation

### DISCUSSION THREAD

**Discussion by seanvosler** [gotenberg/gotenberg#469] [Q&A] [answered]
Exploring: https://gotenberg.dev/docs/modules/pdf-engines

Not seeing details on how to use an engine to convert PDF to MD? Thoughts?

**Comment by gulien** [comment, marked as answer]
Actually, it's the Chromium module that is able to convert from Markdown to PDF (but not vice-versa): https://gotenberg.dev/docs/modules/chromium#markdown.
